backend/src/main/resources/com/bakdata/conquery/models/events/generation/types/VarIntTypeByte.ftl

```diff
@@ -2,6 +2,6 @@
 <#macro kryoSerialization type>output.writeByte(<#nested/>)</#macro>
 <#macro kryoDeserialization type>input.readByte()</#macro>
 <#macro nullCheck type><#nested/> == <@nullValue type=type/></#macro>
-<#macro majorTypeTransformation type>(int)<#nested></#macro>
+<#macro majorTypeTransformation type> ${type.minValue} + (int) <#nested></#macro>
```

Comment:
Muss die Transformation zweimal gemacht werden? Hier und in VarIntTypeByte.java

Reply:
Sind unterschiedliche Stufen aber guter Hinweis